libraries/support/azure/management.rb

```diff
@@ -336,126 +388,128 @@ def sql_database_threat_detection_settings(resource_group, server_name, database
     def sql_database_encryption(resource_group, server_name, database_name)
       get(
         url: link(location: "Microsoft.Sql/servers/#{server_name}/databases/#{database_name}" \
-                            '/transparentDataEncryption/current',
+                          '/transparentDataEncryption/current',
                   resource_group: resource_group),
         api_version: '2014-04-01',
       )
     end
 
-    def webapp(resource_group, webapp_name)
+    def storage_account(resource_group, name)
       get(
-        url: link(location: "Microsoft.Web/sites/#{webapp_name}",
+        url: link(location: "Microsoft.Storage/storageAccounts/#{name}",
                   resource_group: resource_group),
-        api_version: '2016-08-01',
+        api_version: '2017-06-01',
       )
     end
 
-    def webapps(resource_group)
+    def storage_accounts(resource_group)
       get(
-        url: link(location: 'Microsoft.Web/sites',
+        url: link(location: 'Microsoft.Storage/storageAccounts',
                   resource_group: resource_group),
-        api_version: '2016-08-01',
+        api_version: '2017-06-01',
       )
     end
 
-    def webapp_authentication_settings(resource_group, webapp_name)
-      post(
-        url: link(location: "Microsoft.Web/sites/#{webapp_name}" \
-                            '/config/authsettings/list',
-                  resource_group: resource_group),
-        api_version: '2016-08-01',
+    def subnet(resource_group, vnet, name)
+      get(
+        url: link(location: "Microsoft.Network/virtualNetworks/#{vnet}/subnets",
+                  resource_group: resource_group) + name,
+        api_version: '2018-02-01',
       )
     end
 
-    def webapp_configuration(resource_group, webapp_name)
+    def subnets(resource_group, vnet)
       get(
-        url: link(location: "Microsoft.Web/sites/#{webapp_name}/config/web",
+        url: link(location: "Microsoft.Network/virtualNetworks/#{vnet}/subnets",
                   resource_group: resource_group),
-        api_version: '2016-08-01',
+        api_version: '2018-02-01',
       )
     end
 
-    def key_vaults(resource_group)
+    def subscription
       get(
-        url: link(location: 'Microsoft.KeyVault/vaults',
-                  resource_group: resource_group),
-        api_version: '2016-10-01',
+        url: "/subscriptions/#{subscription_id}",
+        api_version: '2019-10-01',
       )
     end
 
-    def key_vault(resource_group, key_vault_name)
+    def subscription_locations
       get(
-        url: link(location: "Microsoft.KeyVault/vaults/#{key_vault_name}",
-                  resource_group: resource_group),
-        api_version: '2016-10-01',
+        url: link(location: 'locations', provider: false),
+        api_version: '2019-10-01',
       )
     end
```

Comment:
@inspec/inspec-cloud-devs  These are the only true additions to this class - the rest is organisation.